<launch>
  <node name="arm_controller" pkg="roomac_arm" type="arm_controller.py" output="screen">
    <rosparam command="load" file="$(find roomac_arm)/config/arm_controller_config.yaml" />
  </node>
</launch>